custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -424,6 +429,7 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
         };
         if (haveAccessToFeatures) {
             taskConfig['beneficiary_category'] = vm.selectedBeneficiaryCategory;
+            taskConfig['thr_report_type'] = vm.selectedBeneficiaryCategory;
```

Comment:
I guess this should be `selectTHRreportType` not `selectedBeneficiaryCategory`

Reply:
good catch!!